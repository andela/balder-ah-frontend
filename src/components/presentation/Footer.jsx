import React from 'react';

const Footer = () => (
  <footer className="page-footer">
    <div className="footer-copyright">
      <div className="container">
        Â© Balder AH
        <a className="grey-text text-lighten-4 right" href="#!">
          Go home
        </a>
      </div>
    </div>
  </footer>
);

export default Footer;
