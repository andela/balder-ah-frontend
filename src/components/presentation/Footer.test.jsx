import React from 'react';
import { render, cleanup } from 'react-testing-library';
import 'jest-dom/extend-expect';
import Footer from './Footer';

afterEach(cleanup);

describe('<Footer />', () => {
  test('should render correctly', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
