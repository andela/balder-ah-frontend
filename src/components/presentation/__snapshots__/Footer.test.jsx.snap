// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<div>
  <footer
    class="page-footer"
  >
    <div
      class="footer-copyright"
    >
      <div
        class="container"
      >
        Â© Balder AH
        <a
          class="grey-text text-lighten-4 right"
          href="#!"
        >
          Go home
        </a>
      </div>
    </div>
  </footer>
</div>
`;
